packages/events-example/src/commands/reconcile/index.ts

```diff
@@ -1,16 +1,8 @@
-import { CliUx, Command, Flags } from '@oclif/core';
+import { Command, Flags } from '@oclif/core';
 import Nile, { Instance, NileApi } from '@theniledev/js';
-import {
-  DestroyResult,
-  InlineProgramArgs,
-  LocalWorkspace,
-  StackSummary,
-  Stack,
-  UpResult,
-  PulumiFn,
-} from '@pulumi/pulumi/automation';
 
-import { pulumiProgram } from '../../pulumiS3';
+import { pulumiProgramGenerator } from '../../pulumiS3';
+import PulumiAwsDeployment from '../../deployments/pulumi';
```

Comment:
it would be good to make this a named export.

eg `import {PulumiAwsDeployment} from '../../deployments`. 

inside `deployments/index.ts` export out `./pulumi`